src/community/ogcapi/ogcapi-changeset/src/main/java/org/geoserver/api/changeset/ChangesetAPICallback.java

```diff
@@ -54,7 +54,7 @@ public void apply(Request dr, OpenAPI api) {
             Parameter fTileParameter = fragment.getComponents().getParameters().get("f-tile");
             api.getComponents().getParameters().put("f-tile", fTileParameter);
             Parameter fParameter = fragment.getComponents().getParameters().get("f-json-zip");
-            api.getComponents().getParameters().put("f-json-zip", fTileParameter);
+            api.getComponents().getParameters().put("f-json-zip", fParameter);
```

Comment:
This is a functional change. I don't understand the code that well, but I'm guessing this is what was intended.

Reply:
Agreed